spec.html

```diff
@@ -17236,6 +17239,19 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. If Type(_rval_) is not Object, throw a *TypeError* exception.
         1. Return ? HasProperty(_rval_, ? ToPropertyKey(_lval_)).
       </emu-alg>
+      <emu-grammar>RelationalExpression : PrivateIdentifier `in` ShiftExpression</emu-grammar>
+      <emu-alg>
+        1. Let _privateIdentifier_ be the StringValue of |PrivateIdentifier|.
+        1. Let _rref_ be the result of evaluating |ShiftExpression|.
+        1. Let _rval_ be ? GetValue(_rref_).
+        1. If Type(_rval_) is not Object, throw a *TypeError* exception.
+        1. Let _env_ be the running execution context's PrivateEnvironment.
+        1. Let _privateNameBinding_ be ? ResolveBinding(_privateIdentifier_, _env_).
+        1. Let _privateName_ be ? GetValue(_privateNameBinding_).
```

Comment:
```suggestion
        1. Let _privateName_ be ! ResolvePrivateIdentifier(_env_, _privateIdentifier_).
```
(I *think* that's right. @Bakkot?)

Also, you might want to rename `_env_` to `_privateEnv_`.

Reply:
Yup. And no particular reason for the assertion afterwards, since `ResolvePrivateIdentifier` always returns a Private Name.